spec/features/ncr/work_orders/approve_spec.rb

```diff
@@ -21,6 +21,21 @@
     end
   end
 
+  scenario "allows delegate to approve work order" do
+    Timecop.freeze do
```

Comment:
do you think it's worth it to try and remove while we are here? It is not clear to me from reading these tests why they would need to set up approvals and observers 10 hours ago.